ydb/core/config/init/init.h

```diff
@@ -315,4 +315,10 @@ class TInitialConfigurator {
     std::unique_ptr<IInitialConfigurator> Impl;
 };
 
+enum class EWorkload {
+    Hybrid /* "hybrid" */,
+    Analyitical /* "analytical" */,
+    Operational /* "operational" */
```

Comment:
Добавь сразу запятую в конец, чтобы при расширении списка не приходилось править существующие строки.
```suggestion
    Operational /* "operational" */, 
```